### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by khanhnguyen1025**

### CODE REVIEW

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+@Component({
+  selector: 'app-edit-profile',
+  templateUrl: './edit-profile.component.html',
+  styleUrls: ['./edit-profile.component.scss']
+})
+export class EditProfileComponent implements OnInit {
+
+
+  // ===>
```
[the remark this answers is not in the document]
Comment (tranmanhlinh) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: Remove the empty lines.

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  @Input() profile: Profile;
+  @Input() fileToUpload: File;
+  @Output() getValidateForm = new EventEmitter<boolean>();
+  // ===>
```
[the remark this answers is not in the document]
Comment (tranmanhlinh) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: Should have access modifiers. (public, private,...)

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  ngOnInit() {
+  }
+
+  handleFileInput(files: FileList) {
+    this.fileToUpload = files.item(0);
+    this.profileService.uploadImage(this.fileToUpload).subscribe();
+  }
+
+  onChangeInput() {
+    console.log(this.profile.sex);
```
Comment (tranmanhlinh) [inline src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: Remove all console.log()

File: src/app/app.component.ts [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
-  logout(): void{
+  logout(): void {
     localStorage.removeItem('currentUser');
     this.router.navigate(['/login']);
   }
-}
 
+  profile(): void {
+    console.log("cocococ");
+
```
Comment (tranmanhlinh) [inline src/app/app.component.ts]: Remove console.log()

File: src/app/pages/profile/components/edit-profile/edit-profile.component.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                (ngModelChange)="onChangeInput()">
+            </nz-input-number>
+        </nz-form-control>
+    </nz-form-item>
+    <nz-form-item>
+        <nz-form-label [nzSpan]="6" nzFor="address">*Gender</nz-form-label>
+        <div nzSpan="14">
+            <nz-select [(ngModel)]="profile.sex" (ngModelChange)="onChangeInput()">
+                <nz-option nzValue="men" nzLabel="Male"></nz-option>
+                <nz-option nzValue="women" nzLabel="Female"></nz-option>
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.html]: done

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+const URL = 'https://helpdesk-kunlez-novahub.herokuapp.com/api/profiles/avatar';
+
+@Component({
+  selector: 'app-edit-profile',
+  templateUrl: './edit-profile.component.html',
+  styleUrls: ['./edit-profile.component.scss']
+})
+export class EditProfileComponent implements OnInit {
+
+
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: done

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: done

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: done

File: src/app/pages/skill/skill.component.ts [hunk 7]
```diff
@@ -51,17 +51,17 @@ export class SkillComponent implements OnInit {
...
+    //   this.listOfData.push({
+    //     id: i,
+    //     name: `Edrward ${i}`,
+    //     category: {
+    //       id: 0,
+    //       name: "Cate 1"
+    //     }
+    //   });
+    // }
+    // this.updateEditCache();
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/skill/skill.component.ts]: done

File: src/app/pages/profile/components/edit-profile/edit-profile.component.ts (hunk 3 above)
Comment (khanhnguyen1025) [reply src/app/pages/profile/components/edit-profile/edit-profile.component.ts]: done

File: src/app/app.component.ts (hunk 4 above)
Comment (khanhnguyen1025) [reply src/app/app.component.ts]: done

**Pull request merged**